elisp/intero.el

```diff
@@ -1791,7 +1837,7 @@ You can always run M-x intero-restart to make it try again.
 (defun intero-get-buffer-create (worker)
   "Get or create the stack buffer for WORKER.
 Uses the directory of the current buffer for context."
-  (let* ((root (intero-project-root))
+  (let* ((root (intero-extend-path-by-buffer-host (intero-project-root)))
```

Comment:
So this will have the effect that the intero worker buffer will be created under `/ssh:foo.com/blah` when proper. That's good.